src/shared/common/components/people.jsx

```diff
@@ -64,15 +64,16 @@ class People extends Component {
                             <td className='govuk-table__cell'>{person.organisation}</td>
                         </tr>
                         }
-                        {person.address.address1 &&
+                        {(person.address.address1 || person.address.address2 || person.address.address3
```

Comment:
This is a change to functionality - should we be adding a test for this component to cover this?